media-libs/sbc/sbc-2.0.recipe

```diff
@@ -12,12 +12,12 @@ COPYRIGHT="2004-2010 Marcel Holtmann
 	2007-2008 Frederic Dalleau"
 LICENSE="GNU GPL v2
 	GNU LGPL v2.1"
-REVISION="1"
+REVISION="2"
 SOURCE_URI="http://www.kernel.org/pub/linux/bluetooth/sbc-$portVersion.tar.gz"
 CHECKSUM_SHA256="897c1463fc6ad25a7822dfa2ce53c340c4ac8054eab381f72aa7d3bf16517738"
 PATCHES="sbc-$portVersion.patchset"
 
-ARCHITECTURES="all"
+ARCHITECTURES="all !x86_gcc2"
 SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
```

Comment:
```suggestion
ARCHITECTURES="all !x86_gcc2"
SECONDARY_ARCHITECTURES="x86"
```
It looks like the `!x86_gcc2` was put on the wrong line to begin with and should already be disabled.

Reply:
Thought it looked weird already :)